chain/client/src/client.rs

```diff
@@ -2094,3 +2106,52 @@ impl Client {
         }
     }
 }
+
+impl Client {
+    fn get_tier1_accounts(&mut self, tip: &Tip) -> anyhow::Result<Arc<AccountKeys>> {
```

Comment:
It should care about network concept, because client configures this network concept. The network is the one which exposes the API, and it doesn't care about which accounts the client considers important at the moment (so it is the other way round). Client is the side wrapping the chain logic, network is the transportation layer. The chain logic determines which messages are critical for the chain progress.

IMO get_tier1_accounts is a good name, because this function returns a value of type from the network API - this function is not applicable in any other context, as it is a centralized place where Client DEFINES what it wants the set of TIER1 accounts to be. In any other context where Client is interested in the set of validators in this and next epoch, it can query the epoch manager via runtime API which is the SoT for this information. Also WRT to readability current_and_incoming_validators requires implicit reasoning that "current" means "current epoch", and "incoming" means "next epoch", which is not that obvious IMO. On the other hand tier1_accounts() is used to populate ChanInfo::tier1_accounts field which is very straightforward.